code/controllers/subsystem/SSambience.dm

```diff
@@ -10,6 +10,10 @@ SUBSYSTEM_DEF(ambience)
 	var/list/ambience_listening_clients = list()
 
 /datum/controller/subsystem/ambience/fire(resumed)
+	if(GLOB.configuration.general.disable_ambient_noise)
```

Comment:
Do this in init please

Reply:
Do the subsystems init after the configuration is loaded? That was one thing I was worried about